app/scripts/directives/blocks.js

```diff
@@ -251,6 +251,14 @@ angular
               return campusNames.data;
             });
         };
+        if ($scope.answer.value) {
+          $scope.searchingCampuses = true;
```

Comment:
Setting a variable just so that it can be checked in a test isn't a pattern I'm used to seeing. Can you remove that line and update the test something like the one I wrote in the other comment?